.github/workflows/ci.yml

```diff
@@ -378,6 +447,7 @@ jobs:
       - name: Generate Landing Page
         working-directory: docs/website
         run: |
+          gcc -v -E -x c++ -isystem/usr/local/share/mrdocs/libcxx/ -isystem/usr/local/share/mrdocs/clang/ -nostdinc++ -nostdlib -
```

Comment:
What's happening here?

Reply:
Just temporary "testing" the defaults include paths give those flags. Removing ...